src/harness/testcases/heartbeat_testcase.py

```diff
@@ -789,23 +771,33 @@ def test_WINNF_FT_S_HBT_17(self):
     self.assertEqual(response['cbsdId'], cbsd_id)
     self.assertEqual(response['response']['responseCode'], 0)
     grant_id = response['grantId']
-
-    # Calculate the Difference Between Current Time and the GrantExpireTime
-    grant_expire_time = time.strptime(response['grantExpireTime'], '%Y-%m-%dT%H:%M:%SZ')
-    current_time = time.strptime(datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%SZ'), '%Y-%m-%dT%H:%M:%SZ')
-    difference_time = time.mktime(grant_expire_time) - time.mktime(current_time)
+    grant_expire_time = datetime.strptime(response['grantExpireTime'], '%Y-%m-%dT%H:%M:%SZ')
     del request, response
 
+    # Calculate the Difference Between Current Time and the GrantExpireTime
+    difference_time = (grant_expire_time - datetime.utcnow()).total_seconds() + 1
     # Log the Wait Time in seconds
-    print "Difference between grantExpireTime and CurrentTime (in seconds)", difference_time
-
+    logging.debug('Difference between grantExpireTime and CurrentTime (in seconds) ', difference_time)
     # Ensure that GrantExpireTime is Greater than CurrentTime
     self.assertGreaterEqual(difference_time, 0)
```

Comment:
Why not use
`self.assertGreaterEqual(grant_expire_time, datetime.utcnow())`

Then the code is so clear that you don't need the comment :)